frontend/templates/product-list/sections/FilterableProductsSection/RefinementSingleSelect.tsx

```diff
@@ -168,8 +168,7 @@ const SingleSelectItem = React.memo(function SingleSelectItem({
                   delete refinementList[attribute];
                }
                return {
-                  ...prevUiState,
-                  refinementList: refinementList,
+                  refinementList,
```

Comment:
```suggestion
                  ...prevUiState,
                  refinementList,
                  page: 0,
```

There may be a number of other things in the UI state. I'm not sure we want to get rid of them all if we're only trying to set the page to 0. Thoughts?

Reply:
Yea I could see an argument for keeping the text in the search box across refinements (which is the only other thing we use the UiState for currently). Does that sound reasonable? @sterlinghirsh

Reply:
The search should definitely remain.

Reply:
Ok, committed this suggestion then.